src/pages/Bridge/utils/findMatchingBridgeAsset.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  BridgeAsset,
+  TokenType as BridgeTokenType,
+} from '@avalabs/bridge-unified';
+
+import {
+  NftTokenWithBalance,
+  TokenType,
+  TokenWithBalance,
+} from '@avalabs/vm-module-types';
+
+export const findMatchingBridgeAsset = (
+  assets: BridgeAsset[],
+  token: Exclude<TokenWithBalance, NftTokenWithBalance>
+): BridgeAsset | undefined => {
+  return assets.find((a) => {
+    if (a.type === BridgeTokenType.NATIVE && token.type === TokenType.NATIVE) {
+      return a.symbol.toLowerCase() === a.symbol.toLowerCase();
```

Comment:
```suggestion
      return a.symbol.toLowerCase() === token.symbol.toLowerCase();
```

Reply:
Good eyes! Fixed in [540a524](https://github.com/ava-labs/core-extension/pull/82/commits/540a5248068a082c2cbf90170d415e5ae968ea32)

Reply:
I also fixed a few other issues and added missing support for dev environment of the avalanche bridge.